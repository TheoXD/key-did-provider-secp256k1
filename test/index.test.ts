import u8a from 'uint8arrays'

import { encodeDID } from '../src'

describe('key-did-provider-secp256k1', () => {
  test('encodeDID', () => {
    const key = u8a.fromString(
      'd713cb7f8624d8648496e01010f2bd72f0dcbbdecdb7036f38c20475f5f429bf',
      'base16'
    )
    expect(encodeDID(key)).toMatchSnapshot()
  })
})
