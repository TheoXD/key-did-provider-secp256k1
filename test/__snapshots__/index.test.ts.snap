// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`key-did-provider-secp256k1 encodeDID 1`] = `"did:key:zQ3shcKCXVrXuW21nQT5zptoEMWTozgoAHvsEGdb7bGBzqqq8"`;
